circuit/algorithms/src/poseidon/prf.rs

```diff
@@ -58,13 +58,15 @@ mod tests {
         let native = console::Poseidon::<<Circuit as Environment>::Network, RATE>::setup(DOMAIN)?;
         let poseidon = Poseidon::<Circuit, RATE>::constant(native.clone());
 
+        let mut rng = TestRng::default();
```

Comment:
Same case as earlier, moving out the rng creation from the function call would speed test cases up